mlton/backend/ssa-to-rssa.fun

```diff
@@ -320,6 +320,30 @@ structure CFunction =
                       return = Type.intInf (),
                       symbolScope = Private,
                       target = Direct (Prim.Name.toString name)}
+
+      val intInfBinary_2Res = fn name =>
+         CFunction.T {args = Vector.new5 (Type.gcState (),
+                                          Type.intInf (),
+                                          Type.intInf (),
+                                          Type.csize (),
+                                          Type.csize ()),
+                      convention = Cdecl,
+                      kind = CFunction.Kind.Runtime {bytesNeeded = SOME 4,  (* CHECK *)
```

Comment:
The `bytesNeeded = SOME 4` field indicates that argument at index 4 holds the number of bytes required to be available on the by the primitive.  The limit check insertion pass makes sure that a heap limit check is performed using that argument to ensure that the required number of bytes are available.  Hence, you need to compute the total number of heap bytes required by IntInf_quotRem (the vector, the quot, and the rem) into one argument and make that the `bytesNeeded` argument.

Reply:
Ok.  I'll make the revision that you suggested with 3 csize () args, since the number of bytes needed for each argument (or at least the first) are required explicitly as well.

Reply:
You could recompute the required limbs of the quot on the C side by examining the arguments, but I agree that would be duplicating work.